actions/create_instance.yaml

```diff
@@ -30,6 +30,16 @@ parameters:
     type: "string"
     description: "SSH key to use during intial instance creation"
     required: false
+  BlockDeviceMappings:
```

Comment:
Do we need to default this to an empty array, same as we do for ``SecurityGroupIds`` so the workflow won't fail?